js/flickity.js

```diff
@@ -752,6 +752,25 @@ proto.queryCell = function( selector ) {
   return this.getCell( selector );
 };
 
+proto.checkVisibility = function() {
+  var viewportX = this.viewport.getBoundingClientRect().x;
+  var viewportWidth = this.viewport.offsetWidth;
+
+  this.cells.forEach(function (cell) {
+    var cellX = cell.element.getBoundingClientRect().x - viewportX;
+    var isVisible = (
```

Comment:
In my testing, `isVisible` will be false when the `is-selected` cell is itself wider than the browser window, despite that the cell is obviously visible (being the `is-selected` cell centered on screen).